numba/hsa/tests/hsadrv/test_driver.py

```diff
@@ -119,6 +125,272 @@ def test_allocate(self):
             self.assertEqual(ptr[i], src[i])
         hsa.hsa_memory_free(ptr)
 
+    def apu_present():
```

Comment:
Since apu_present() and dgpu_count() are not methods, you can put them at the top-level (Python is not Java :-)).